openpype/hosts/maya/plugins/publish/extract_fbx_animation.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+import os
+
+from maya import cmds  # noqa
+import pyblish.api
+
+from openpype.pipeline import publish
+from openpype.hosts.maya.api import fbx
+
+
+class ExtractFBXAnimation(publish.Extractor):
+    """Extract Rig in FBX format from Maya.
+
+    This extracts the rig in fbx with the constraints
+    and referenced asset content included.
+    This also optionally extract animated rig in fbx with
+    geometries included.
+
+    """
+    order = pyblish.api.ExtractorOrder
+    label = "Extract Animation (FBX)"
+    hosts = ["maya"]
+    families = ["animation.fbx"]
+
+    def process(self, instance):
+        # Define output path
+        staging_dir = self.staging_dir(instance)
+        filename = "{0}.fbx".format(instance.name)
+        path = os.path.join(staging_dir, filename)
+
+        # The export requires forward slashes because we need
+        # to format it into a string in a mel expression
+        fbx_exporter = fbx.FBXExtractor(log=self.log)
+        out_set = instance.data.get("animated_skeleton", [])
+        # Export
+        instance.data["constraints"] = True
+        instance.data["skeletonDefinitions"] = True
+        instance.data["referencedAssetsContent"] = True
+
+        fbx_exporter.set_options_from_instance(instance)
+
+        out_set_name = next(out for out in out_set)
+        # temporarily disable namespace
+        namespace = out_set_name.split(":")[0]
+        new_out_set = out_set_name.replace(
+            f"{namespace}:", "")
+        cmds.namespace(set=':')
+        cmds.namespace(set=namespace)
+        cmds.namespace(rel=True)
```

Comment:
Use the long argument name here since `rel` is a bit ambiguous when reading it. Assume people don't know maya's API that well that they know all these flags ;)
```suggestion
        cmds.namespace(relativeNames=True)
```